src/main/java/net/nuagenetworks/bambou/RestSession.java

```diff
@@ -203,10 +203,15 @@ public void instantiateChild(RestObject restObj, RestObject childRestObj, RestOb
 
     @Override
     public void assign(RestObject restObj, List<? extends RestObject> childRestObjs) throws RestException {
-        restObj.assign(this, childRestObjs);
+        assign(restObj, childRestObjs, null);
```

Comment:
For consistency, this line should read: restObj.assign(this, childRestObjs) or restObj.assign(this, childRestObjs, null);